spec/lib/inclusion_intersection_validator_spec.rb

```diff
@@ -24,7 +24,7 @@ class BadSetupValidator
   class ListValidator
     include ActiveModel::Validations
     attr_accessor :list
-    validates_with ActiveModelValidatesIntersectionOf::Validator, attributes: [:list], in: VALID_ARRAY, message: "not valid"
+    validates_with ActiveModelValidatesIntersectionOf::Validator, attributes: [:list], within: VALID_ARRAY, message: "not valid"
   end
 
   subject { List.new }
```

Comment:
Wow!! When I was reviewing your code, I just figure out that here is not `subject { List.new }` but `subject { ListValidator.new }`

Can you update this for me, with the other review comments?